playgrounds/petclinic-maven-sb-2/src/main/scss/typography.scss

```
@@ -0,0 +1,60 @@
+
+  src: url('../fonts/varela_round-webfont.eot');
+  src: url('../fonts/varela_round-webfont.eot?#iefix') format('embedded-opentype'),
+       url('../fonts/varela_round-webfont.woff') format('woff'),
+       url('../fonts/varela_round-webfont.ttf') format('truetype'),
+       url('../fonts/varela_round-webfont.svg#varela_roundregular') format('svg');
+  font-weight: normal;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: 'montserratregular';
```

Stylelint (17.14.0)

[error] Expected no quotes around "montserratregular" (font-family-name-quotes)

(font-family-name-quotes)